Library/Formula/sshuttle.rb

```diff
@@ -1,8 +1,8 @@
 class Sshuttle < Formula
   desc "Proxy server that works as a poor man's VPN"
   homepage "https://github.com/sshuttle/sshuttle"
-  url "https://github.com/sshuttle/sshuttle/archive/sshuttle-0.72.tar.gz"
-  sha256 "3ea217fa98e3887b0fb2229eb65b9548a4beea1947ed5949501e5ceec360ba19"
+  url "https://github.com/sshuttle/sshuttle/archive/v0.73.tar.gz"
+  sha256 "d78a6aa76f93ab69fd9685ce11b428a9b549c049a6c0104740c06c9b354a5ae3"
   head "https://github.com/sshuttle/sshuttle.git"
 
   bottle :unneeded
```

Comment:
Should have a `depends_on :python if MacOS.version <= :snow_leopard` if it's using Python now.